build.js

```diff
@@ -626,6 +626,9 @@ const buildIDLMemberTests = (
     }
 
     const isStatic = member.special === 'static' || iface.type === 'namespace';
+    const isEventHandler =
+      member.idlType?.type === 'attribute-type' &&
+      (member.idlType?.idlType || '').includes('EventHandler');
```

Comment:
Why includes? Can it be some other type than exactly "EventHandler"?

Reply:
That's correct, I've noticed some special names for event handlers for certain interfaces.

Reply:
Hrm, this seems to have revealed a bug I wasn't expecting to see...